src/main/resources/application.yaml

```diff
@@ -1,23 +1,50 @@
+database:
+  host: ${DATABASE_HOST:jdbc:mysql://localhost:3306}
+  name: ${DATABASE_NAME:notification}
+  use-ssl: ${DATABASE_USESSL:false}
 server:
   port: 9006
-
+info:
+  app:
+    name: "Notification Service"
+    description: "API for sending notifications"
 logging:
   pattern:
     console: "%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n"
   level:
-    root: ${ROOT_LOGGING_LEVEL:INFO}
+    root: ${ROOT_LOGGING_LEVEL:DEBUG}
     org.apache.http: WARN
 
 govNotify:
   key: ${GOV_NOTIFY_API_KEY:key}
 
 oauth:
   serviceUrl: ${OAUTH_SERVICE_URL:http://localhost:8080}
-  clientId: ${CLIENT_ID:9fbd4ae2-2db3-44c7-9544-88e80255b56e}
-  clientSecret: ${CLIENT_SECRET:test}
-  checkTokenEndpoint: ${OAUTH_CHECK_TOKEN_ENDPOINT:/oauth/check_token}
-  checkTokenUrl: "${oauth.serviceUrl}${oauth.checkTokenEndpoint}"
+#  checkTokenEndpoint: ${OAUTH_CHECK_TOKEN_ENDPOINT:/oauth/check_token}
+#  checkTokenUrl: "${oauth.serviceUrl}${oauth.checkTokenEndpoint}"
   tokenEndpoint: ${OAUTH_TOKEN_ENDPOINT:/oauth/token}
```

Comment:
Could we please default these token endpoints to oAuth2 as per the new identity-service.